gce_image_import_export_tests.Dockerfile

```diff
@@ -11,27 +11,30 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
-FROM golang:alpine
-
-RUN apk add --no-cache git
+FROM golang
 
 # Build test runner
-WORKDIR /
-RUN go get -d github.com/GoogleCloudPlatform/compute-image-tools/gce_image_import_export_tests
-RUN CGO_ENABLED=0 go build -o /gce_image_import_export_test_runner github.com/GoogleCloudPlatform/compute-image-tools/gce_image_import_export_tests
+WORKDIR /gce_ovf_import_tests
```

Comment:
why gce_ovf_import_tests?

Shouldn't it be gce_image_import_export_tests?

Reply:
Yep, this was a typo